docs/security.md

```diff
@@ -0,0 +1,15 @@
+# Dasharo Security
+
+## Reporting security issues in Dasharo
+
+!!! note "Please note"
+
+    Dasharo security email address is intended for responsible disclosure by
+    security researchers and others who discover legitimate security
+    vulnerabilities. Please do not attempt to contact us via this address unless
+    you can demonstrate an actual security vulnerability or provide us with
+    reasonable steps we could follow to verify your claims.
+
+If you’ve discovered a security issue affecting Dasharo, please send en
+encrypted (using [Dasharo security team PGP key](TBD)) e-mail to this address:
```

Comment:
Discussed offline with @pietrushnic that he will create such key into secpack

Reply:
Maybe worth adding an info that unencrypted mails will be ignored? It may reduce the noise.

Reply:
@pietrushnic can you please handle the key generation?

Reply:
This should be resolved since 2nd Oct.